config/bedrock/config.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+//
+// SPDX-License-Identifier: CC0-1.0
+
+package bedrock
+
+import (
+	"github.com/crossplane/upjet/pkg/config"
+	"github.com/upbound/provider-aws/config/common"
+)
+
+// Configure adds configurations for the bedrock group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_bedrock_model_invocation_logging_configuration", func(r *config.Resource) {
+		r.References["logging_config.s3_config.bucket_name"] = config.Reference{
+			Type: "github.com/upbound/provider-aws/apis/s3/v1beta1.Bucket",
+		}
+		r.References["logging_config.cloudwatch_config.role_arn"] = config.Reference{
+			Type:      "github.com/upbound/provider-aws/apis/iam/v1beta1.Role",
+			Extractor: common.PathARNExtractor,
+		}
```

Comment:
In order for [SingleNestedBlocks](https://github.com/hashicorp/terraform-provider-aws/blob/4ec8ee85f8175b86a217136c120b667a822f481a/internal/service/bedrock/model_invocation_logging_configuration.go#L46) to be generated properly, we need to manually add the following configurations.

Similar example: https://github.com/turkenf/provider-aws/blob/997e72c93954493710b4327098dfbc1928969063/config/opensearchserverless/config.go#L13

```
		r.SchemaElementOptions.SetEmbeddedObject("logging_config")
		r.SchemaElementOptions.SetEmbeddedObject("logging_config.s3_config")
		r.SchemaElementOptions.SetEmbeddedObject("logging_config.cloudwatch_config")
```